tests/.vscode/launch.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
launch.json file is being checked in for developers who may need to add new test case or debug existing code using visual studio. this file has the settings necessary to debug python file. one just needs to make sure this folder ".vscode" exists at the same location where the tests folder is